### PR TITLE
Remove usage of deprecated APIs

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

Or mark their use as expected

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [x] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: http/http-advanced-reactive/deps.tree [hunk 1]
```diff
@@ -0,0 +1,416 @@
+[INFO] Scanning for projects...
```
Comment (michalvavrik) [inline http/http-advanced-reactive/deps.tree:1]: Why are you adding this file?

**Review by michalvavrik — COMMENTED** [review]
Changes LGTM, I'll need to understand `deps.tree` addition and when that is handled, I'll also run OCP although I think chance it will have related changes is very little.

File: http/http-advanced-reactive/deps.tree (hunk 1 above)
Comment (fedinskiy) [reply http/http-advanced-reactive/deps.tree]: by mistake

**Comment by fedinskiy** [comment]
run tests

**Review by michalvavrik — APPROVED** [review]
LGTM, thx!

**Pull request merged**